src/Meilisearch/Autocomplete/Configuration/Source.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusMeilisearchPlugin\Meilisearch\Autocomplete\Configuration;
+
+final class Source
+{
+    public function __construct(
+        public readonly string $id,
+        public readonly string $index,
+        /**
+         * The attribute that holds the URL on any given item
+         */
```

Comment:
Imo the comment is not needed, but if you insist it to be here, I would at least make it one-liner 🖖

Reply:
One-lined :D About comments, I believe more is more :D